test/Microsoft.Identity.Abstractions.Tests/AquireTokenOptionsTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Xunit;
+
+namespace Microsoft.Identity.Abstractions.Tests
+{
+    public class AquireTokenOptionsTests
+    {
+        [Fact]
+        public void ManagedIdentitySystemAssigned()
+        {
+            // App token from a system-assigned managed identity.
+            // -------------------------------------------------
+            // https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview
+            /*
+            // <managedidentity_json>
+            {
+                "AquireTokenOptions": {
+                    "ManagedIdentity": {
+                        "IdType": "SystemAssigned"
+                    }
+                }
+            }
+            // </managedidentitysystem_json>
+            */
+
+            // <managedidentity_csharp>
+            ManagedIdentityDescription managedIdentityDescription = new ManagedIdentityDescription
+            {
+                IdType = ManagedIdentityType.SystemAssigned
+            };
+
+            AcquireTokenOptions acquireTokenOptions = new AcquireTokenOptions
+            {
+                ManagedIdentity = managedIdentityDescription
+            };
+            // </managedidentitysystem_csharp>
+
+            Assert.Equal(ManagedIdentityType.SystemAssigned, acquireTokenOptions.ManagedIdentity.IdType);
+            Assert.Null(acquireTokenOptions.ManagedIdentity.IdValue);
+        }
+
+        [Fact]
+        public void ManagedIdentityUserAssigned()
+        {
+            // App token from a user-assigned managed identity.
+            // -------------------------------------------------
+            // https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview
+            /*
+            // <managedidentityuser_json>
+            {
+                "AquireTokenOptions": {
+                    "ManagedIdentity": {
+                        "IdType": "UserAssignedClientId"
+                        "IdValue": "[InsertRelevantValueForTheManagedIdentityResource]"
```

Comment:
sorry, I'm still not understanding what `IdType` and `IdValue` are.

Reply:
design changed no longer relevant